plugin.video.curiositystream/main.py

```diff
@@ -0,0 +1,205 @@
+import functools
+import sys
+from contextlib import contextmanager
+from urlparse import parse_qsl
```

Comment:
not python3 compatible, you might want to make it py2 and py3 compatible to ease the migration and future submission to the matrix branch

Reply:
I've used try/expect but not sure if it's the best practice in that cases. Please, let me know

Reply:
```python
import sys

PY3 = sys.version_info.major

if PY3:
    your py3 import logic
else:
   your py2 import logic
```

Reply:
Thanks :-) Done 👍